src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -1093,86 +1099,82 @@ export function getAlertContent(alert) {
+            <p>In order to participate, you must return the RAMP Opt-in Election form. If you choose to participate in RAMP, VA will withdraw all of your eligible appeals and instead review your case using the lane you select. If you do not want to participate in RAMP and wish to continue your appeal under the existing process, no action is required.</p>
           </div>
         ),
         displayType: 'info',
         type,
       };
     case ALERT_TYPES.rampIneligible:
       return {
-        title: 'Your appeal is not eligible for the Rapid Appeals Modernization Program (RAMP)',
+        title: 'This appeal is not eligible for the Rapid Appeals Modernization Program',
         description: (
-          <p>This appeal is not eligible for RAMP because it’s near the front
-          of the docket line and ready to be assigned to a Veterans Law Judge
-          at the Board. For more information, please call your VSO or
-          representative.</p>
+          <p>On {formattedDate}, VA sent you a letter informing you of a new program called the Rapid Appeals Modernization Program (RAMP). However, this appeal is not eligible to participate in RAMP because it {statusDescription}. If you have other appeals, they may still be eligible to participate in RAMP.</p>
         ),
         displayType: 'info',
         type,
       };
     case ALERT_TYPES.decisionSoon:
       return {
-        title: 'Your appeal is near the front of line',
+        // TODO: confirm which of duplicate content is correct
+        title: 'Decision soon',
         description: (
-          <p>Your appeal is near the front of the Board’s docket line for a
-          Veterans Law Judge to review your appeal and write a decision.
-          Sending in new evidence at this time will delay your case being
-          reviewed. Please make sure the Board has your correct mailing address.</p>
+          <p>Your appeal will soon receive a Board decision. Sending in new evidence at this time could delay review of your appeal. If you’ve moved recently, please make sure that VA has your current mailing address.</p>
         ),
         displayType: 'info',
         type,
       };
     case ALERT_TYPES.blockedByVso:
       return {
-        title: 'Waiting on your representative',
+        title: 'A judge cannot review your appeal',
+        // TODO: confirm vsoName variable
         description: (
-          <p>You’re at the front of the Board’s docket line, but your appeal is
-          with your Veteran Service Organization. Please contact them to make
-          sure they are sending the needed information to the Board.</p>
+          <p>Your appeal is eligible to be assigned to a judge based on its place in line, but they are prevented from reviewing your appeal because your Veteran Service Organization, {details.vsoName}, is currently reviewing it. For more information, please contact your Veteran Service Organization or representative.</p>
```

Comment:
Is `vsoName` the correct thing to use here?

Reply:
Yes. And if you want to check what the attribute names in the details objects, check out this issue: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/7023

Reply:
Looking at [this issue](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/7023), `vsoName` looks right. :+1:

@cmgiven Can you confirm that it's display-ready without any transformation?

Reply:
It is display ready, yes.

Reply:
Thanks @cvalarida @cmgiven.